src/test/kotlin/no/nav/syfo/melding/kafka/KafkaDialogmeldingFraBehandlerConsumerSpek.kt

```diff
@@ -37,6 +47,157 @@ class KafkaDialogmeldingFraBehandlerSpek : Spek({
                     }
 
                     verify(exactly = 1) { mockConsumer.commitSync() }
+
+                    database.getMeldingerForArbeidstaker(UserConstants.ARBEIDSTAKER_PERSONIDENT).size shouldBeEqualTo 0
+                }
+                it("Receive dialogmelding DIALOG_SVAR but unknown conversationRef") {
+                    val dialogmelding = generateDialogmeldingFraBehandlerDTO(
+                        uuid = UUID.randomUUID(),
+                        msgType = DialogmeldingType.DIALOG_SVAR.name,
+                    )
+                    val mockConsumer = mockKafkaConsumerWithDialogmelding(dialogmelding)
+
+                    runBlocking {
+                        pollAndProcessDialogmeldingFraBehandler(
+                            kafkaConsumerDialogmeldingFraBehandler = mockConsumer,
+                            database = database,
+                        )
+                    }
+
+                    verify(exactly = 1) { mockConsumer.commitSync() }
+
+                    database.getMeldingerForArbeidstaker(UserConstants.ARBEIDSTAKER_PERSONIDENT).size shouldBeEqualTo 0
+                }
+                it("Receive dialogmelding DIALOG_SVAR and known conversationRef") {
+                    val dialogmeldingSendt = generateMeldingTilBehandlerRequestDTO().toMeldingTilBehandler(
+                        personident = UserConstants.ARBEIDSTAKER_PERSONIDENT,
+                    )
+                    database.connection.use {
+                        it.createMeldingTilBehandler(dialogmeldingSendt)
+                    }
+                    val pMeldingListBefore = database.getMeldingerForArbeidstaker(UserConstants.ARBEIDSTAKER_PERSONIDENT)
+                    pMeldingListBefore.size shouldBeEqualTo 1
+                    val pMelding = pMeldingListBefore.first()
+                    pMelding.uuid shouldBeEqualTo dialogmeldingSendt.uuid
+                    pMelding.arbeidstakerPersonIdent shouldBeEqualTo dialogmeldingSendt.arbeidstakerPersonIdent.value
+                    pMelding.innkommende shouldBe false
```

Comment:
Trenger man dette? Her sjekker man jo bare at `createMeldingTilBehandler` fungerer, og det kan jo evt. ligge i en egen test hvis man trenger det?

Reply:
Ja, men poenget her er at vi må få opprettet en sendt forespørsel for å sjekke at vi klarer å ta i mot et svar.